src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -825,45 +850,24 @@ PaError PaPulseAudio_StartStreamCb( PaStream * s )
             {
                 PaPulseAudio_Lock( pulseaudioHostApi->mainloop );
 
-                if ( ! pa_stream_connect_playback( stream->outputStream,
+                if ( pa_stream_connect_playback( stream->outputStream,
                                                    pulseaudioName,
                                                    &stream->outputBufferAttr,
                                                    pulseaudioStreamFlags,
                                                    NULL,
                                                    NULL ) )
                 {
-                    pa_stream_set_underflow_callback( stream->outputStream,
-                                                      PaPulseAudio_StreamUnderflowCb,
-                                                      stream);
-                    pa_stream_set_started_callback( stream->outputStream,
-                                                    PaPulseAudio_StreamStartedCb,
-                                                    stream );
-                }
-                else
-                {
+                    PA_PULSEAUDIO_SET_LAST_HOST_ERROR( 0,
```

Comment:
Still broken here. `PA_PULSEAUDIO_SET_LAST_HOST_ERROR` must only be used when returning `paUnanticipatedHostError` which is not the case here. Also, passing 0 to `PA_PULSEAUDIO_SET_LAST_HOST_ERROR` makes no sense, that is not a pulseaudio error code. You should be passing the error code from pulse audio.